### PR TITLE
Update configuration.py

### PR DESCRIPTION
**Pull request by rahulg963**
Problem in import statement of line 6.

### CODE REVIEW

File: cachet_url_monitor/configuration.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import logging
 import time
 
-import cachet_url_monitor.status
+import status as st
+#import st
```
Comment (mtakaki) [inline cachet_url_monitor/configuration.py]: Could you please remove this commented-out line?

**Comment by mtakaki** [comment]
For some reason codeship build didn't run. I don't even see it started on their console. I'll try kicking off a build to check if your changes will pass.

**Comment by rahulg963** [comment]
Sure I will remove the comment.

**Comment by mtakaki** [comment]
Closing this one in favor of #13

**Pull request closed without merging**